### ISSUE TITLE
Protobuf converter for backwards compatibility

### ISSUE THREAD

**Issue by cgebbe** [tensorflow/tensorflow#29795]
**System information**
- TensorFlow version (you are using): 1.3.0 / 1.13.1
- Are you willing to contribute it (Yes/No): yes

**Describe the feature and the current behavior/state.**
Situation: I need to develop a CNN for a system which only has tensorflow 1.3.0. Since that version doesn't come with keras, I simply create a model using tensorflow 1.13.1 and save it as a protobuf file. Then, I import the protobuf file in tensorflow 1.3.0

Expected result: I can import the protobuf file also in tensorflow 1.3.0, since I didn't use any new features

Actual result: I get an error similar to https://github.com/tensorflow/tensorflow/issues/17628

**Proposed solution: Can we write a converter, which tries to translate protobufs file generated from newer tensorflow versions to ones which can be read in older versions?**

**Will this change the current api? How?**
This could be an additional script, but would not change existing code

**Who will benefit with this feature?**
Anybody who has to develop models for systems running old tensorflow versions

**Any Other info.**
In order to start one would need an overview of how the protobuf file generation (from graph_defs) has developed in each tensorflow version. Is there any such overview available?

**Comment by jvishnuvardhan** [comment]
@cgebbe Sorry for the delay in my response. There were huge improvements in TF. Why do you want to use TF1.3? Do you have any use case that requires TF1.3 which was not associated with keras? Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by jvishnuvardhan** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by cgebbe** [comment]
The issue came up because Udacity only features tensorflow 1.3.0 in their final capstone project of their self-driving car nanodegree, see https://github.com/udacity/CarND-Capstone/issues/276

I thought that such situations may occur quite often. However, I don't have the issue anymore, so it can be closed.
